code/_onclick/hud/plane_master_controller.dm

```diff
@@ -87,4 +87,4 @@ INITIALIZE_IMMEDIATE(/atom/movable/plane_master_controller)
 
 /atom/movable/plane_master_controller/game
 	name = PLANE_MASTERS_GAME
-	controlled_planes = list(FLOOR_PLANE, GAME_PLANE, LIGHTING_PLANE)
+	controlled_planes = list(FLOOR_PLANE, GAME_PLANE, LIGHTING_PLANE, PSYCHIC_PLANE)
```

Comment:
Shouldn't it get its own plane controller? Why is psychic plane controller by master controller?